openghg/store/_obssurface.py

```diff
@@ -503,7 +509,10 @@ def store_data(
         # Very rudimentary hash of the data and associated metadata
         hashes = hash_retrieved_data(to_hash=data)
         # Find the keys in data we've seen before
```

Comment:
Could we rename `seen_before` to something like `file_hashes_to_compare` given this new check and update the comment. I just think otherwise the subtly here of what is being checked may not be clear.

Reply:
Done